### ISSUE TITLE
Problem with Gh-Pages Deployment

### ISSUE THREAD

**Issue by vivekkushalch** [shahank42/dwitter#1]
[https://shahank42.github.io/dwitter](https://shahank42.github.io/dwitter)

**No Resources were loaded, resulting in a Blank Page.**

**Comment by shahank42** [comment]
Yeah I know. I'll have a look into they deployment once I get back to this project.

**Comment by vivekkushalch** [comment]
Okay.
